x-pack/plugins/secops/public/pages/hosts/host_details.tsx

```diff
@@ -7,26 +7,67 @@
 import {
   // @ts-ignore: EuiBreadcrumbs has no exported member
   EuiBreadcrumbs,
-  EuiPanel,
-  EuiTitle,
+  EuiFlexGroup,
 } from '@elastic/eui';
+import { isUndefined } from 'lodash';
 import React from 'react';
 import { pure } from 'recompose';
 import styled from 'styled-components';
+import chrome from 'ui/chrome';
+import { EmptyPage } from '../../components/empty_page';
 import { getHostsUrl, HostComponentProps } from '../../components/link_to/redirect_to_hosts';
+import { HostDetailsPanel } from '../../components/page/hosts/hosts_details_panel';
+import { manageQuery } from '../../components/page/manage_query';
+import { GlobalTime } from '../../containers/global_time';
+import { HostsQuery } from '../../containers/hosts';
+import { WithSource } from '../../containers/source';
+import * as i18n from './translations';
+
+const basePath = chrome.getBasePath();
+
+const HostDetailsPanelManage = manageQuery(HostDetailsPanel);
 
 export const HostDetails = pure<HostComponentProps>(({ match }) => (
-  <>
-    <HostBreadcrumbWrapper breadcrumbs={getBreadcrumbs(match.params.hostId!)} />
-    <EuiPanel>
-      <EuiTitle>
-        <h2>Host Details</h2>
-      </EuiTitle>
-      <div>
-        Match Params: <pre>{JSON.stringify(match, null, 2)}</pre>
-      </div>
-    </EuiPanel>
-  </>
+  <WithSource sourceId="default">
+    {({ auditbeatIndicesExist }) =>
+      auditbeatIndicesExist || isUndefined(auditbeatIndicesExist) ? (
```

Comment:
Is the intent of the second clause of the condition

`|| isUndefined(auditbeatIndicesExist)`

to make a request when we don't know whether or not the indices exist?

Reply:
I was following convention from the main hosts page: https://github.com/elastic/kibana/blob/77e91bc25577fbf5aec470bfcb38c69957e19798/x-pack/plugins/secops/public/pages/hosts/hosts.tsx#L42

but, yeah, that's my understanding. I was curious though if this should be hoisted up to the [HostsContainer](https://github.com/elastic/kibana/blob/77e91bc25577fbf5aec470bfcb38c69957e19798/x-pack/plugins/secops/public/pages/hosts/index.tsx) as we're not currently showing separate errors per page?

Reply:
As per our conversation and clarification with Xavier, created an appropriately named function `indiciesExistOrDataTemporarilyUnavailable()` alongside the `WithSource` component and updated to use it here and within the main `Hosts` page as well.

Also discussed, for my follow-up, no need to hoist this up to the `HostsContainer` as its chief concern is routing.